app/src/main/java/com/framgia/englishconversation/data/source/remote/comment/CommentRepository.java

```diff
@@ -41,7 +42,17 @@ public void removeListener() {
     }
 
     @Override
-    public void saveRevisionComment(Comment comment) {
-        mDataSource.saveRevisionComment(comment);
+    public Observable<Comment> saveRevisionComment(Comment comment) {
+       return mDataSource.saveRevisionComment(comment);
+    }
+
+    @Override
+    public Observable<Comment> deleteComment(Comment comment) {
+        return mDataSource.deleteComment(comment);
+    }
+
+    @Override
+    public Observable<Comment> saveCommentDelete(Comment comment) {
+      return   mDataSource.saveCommentDelete(comment);
```

Comment:
ERROR: 'method def' child have incorrect indentation level 6, expected level should be 8.